pkg/lib/cockpit-components-dialog.jsx

```diff
@@ -237,6 +223,9 @@ export class Dialog extends React.Component {
                 </Button>
             </Popover>;
 
+        const error_alert = this.props.error &&
+            <Alert variant='danger' isInline title={React.isValidElement(this.props.error) ? this.props.error : this.props.error.toString() }>{this.props.error.details}</Alert>;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17221-20221202-001251-c9401003-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)